pkg/smart/parse.go

```diff
@@ -245,36 +257,40 @@ func smartCtlParse(raw []string) (common.MeasurementsMap, []error) {
 	return marshaledDisks, errs
 }
 
-func smartctlIsSupportedVersion(buildStr string) error {
+func smartctlIsSupportedVersion(buildStr string) (string, error) {
```

Comment:
the function name is kind of confusing.
It's actually confusing because it makes two things:
1) parsing version string
2) checking for compatibility
Splitting the function will simplify the code

Reply:
done